.github/workflows/validate-schema.yml

```diff
@@ -0,0 +1,65 @@
+name: Schema
+# Validate schemas modified in an pull request or push against their given meta-schema (such as draft-07, etc.)
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - au.org.access-nri/**.json
+  pull_request:
+    branches:
+      - main
+    paths:
+      - au.org.access-nri/**.json
+jobs:
+  setup:
+    name: Setup
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.changed.outputs.all_changed_and_modified_files }}
```

Comment:
Sorry if a silly question but why `all_changed_and_modified_files` vs `all_changed_files`?

Reply:
It was more permissive (Added, Copied, Modified, Deleted, Renamed,Type changed, Unmerged, Unknown VS Added, Copied, Modified, Renamed) but I realise that I don't need to keep track of all those events, thanks! Done in https://github.com/ACCESS-NRI/schema/pull/37/commits/c8f6a014d08dab89c86a51fa01bbbc67e8bd8e46